website/project/Contributors.scala

```diff
@@ -22,15 +22,18 @@ object Contributors {
     JavaNetClientBuilder[IO](blocker).create // use BlazeClientBuilder for production use
   }
 
-  val token: Option[String] = sys.env.get("GITHUB4S_ACCESS_TOKEN")
+  val token: Option[String] = sys.env.get("GITHUB_TOKEN")
 
   def contributors(repo: GitHubRepository): List[User] =
     Github[IO](httpClient, token)
       .repos
       .listContributors(repo.owner, repo.repo)
       .unsafeRunSync()
       .result match {
-        case Left(e) => throw new Exception(s"Unable to fetch contributors for ${repo.serialize}. Did you misspell it? Did the repository move?")
+        case Left(e) => throw new Exception(
```

Comment:
do we really need this to be a hard error? Could we just stick something in like "failed to grab the contributors for <...>, will try again on next website rebuild!"

Reply:
That shouldn't be necessary now that we will properly be using the token. Besides, if we did make it silently succeed and we broke it, we would never notice because odds are we'll never look and see that warning.

Reply:
yeah your fix is better